### PR TITLE
HOCS-2423 : DCU draft teams

### PR DESCRIPTION
**Pull request by MarkWrightBjss**
Redirect drafting teams request to new endpoint that gets teams by stage type.

### CODE REVIEW

File: server/lists/index.js [hunk 1]
```diff
@@ -293,7 +293,7 @@ module.exports = {
         },
         DRAFT_TEAMS: {
             client: 'INFO',
-            endpoint: '/teams/drafters',
```
Comment (zuzak) [inline server/lists/index.js:296]: Is there a `/teams/drafters` endpoint we should tear out?

Comment (zuzak) [reply server/lists/index.js:296]: ah, you already have. Don't mind me

**Review by philbarton — APPROVED** [review]
Looks good to me

File: server/lists/index.js (hunk 1 above)
Comment (MarkWrightBjss) [reply server/lists/index.js:296]: You're absolutely right. See here:
https://github.com/UKHomeOffice/hocs-info-service/pull/306/files

**Review by SeanReedHomeOffice — APPROVED** [review]
LGTM

**Review by SeanReedHomeOffice — APPROVED** [review]
Maybe should be called DCU_DRAFT_TEAMS to avoid confusion. But not a biggie

**Pull request merged**